e2e/README.md

```diff
@@ -0,0 +1,39 @@
+# Guide to Running Tests
+
+It is important to note that one must ensure that all dfusion related builds are recent (i.e. driver, graph-listener and their dependents). 
+
+From within the project source run:
+
+```shell script
+docker-compose down && docker-compose up driver graph-listener truffle
```

Comment:
we should probably use the same testing procedure as the Travis file, as that is what we test:
```
docker-compose down
./scripts/setup-contracts.sh
docker-compose up -d driver graph-listener
```

Doing it this way also makes it so there are WAY less errors in both the driver and listener.

Reply:
~~I am sorry, but this is simply not possible for those users who are on macOS. Unless it would make sense to split the guide into OS-specific instructions (or at least include a phrase in the readme about why the test guide differs from travis. I believe this has somethign to do with the fact that Docker does not run natively in darwin, but instead inside its own container. Then there is some problem running binaries etc... @fleupold might be able to elaborate on this.~~

Oh! You are proposing that we use the Guide instructions in Travis instead of what is currently there! Really really sweet idea. I think its also much easier anyway. However, if I recall, the reason for the special (and more complicated) Travis execution was for build speed improvement. Is this right @fleupold?

Reply:
I would argue (at least locally) that the three lines proposed here are equivalent to 

```
docker-compose down && docker-compose up driver graph-listener truffle
```

since `truffle` container is essentially the same as `./scripts/setup-contracts.sh`

I wonder if we could make this work...

Reply:
I think I will save this suggestion for a separate PR. Particularly I will address this in #474 when reworking the travis file (if this is ok with you).

Reply:
Sure, my only concern was that if the contracts were not deployed on a testnet before building, then you get build errors for the driver and listener.

Reply:
Sure, my only concern was that if the contracts were not deployed on a testnet before building, then you get build errors for the driver and listener.